Model/reviews.post.model.js

```diff
@@ -0,0 +1,47 @@
+const db = require("../db/connection.js");
+
+exports.selectReviewPost = (review_id, patchInfo) => {
+  const toPatchThisKey = Object.keys(patchInfo);
+  const validReviewKeys = [
+    "title",
+    "review_body",
+    "designer",
+    "review_img_url",
+    "votes",
+    "category",
+    "owner",
+  ];
+  return db
+    .query(`SELECT * FROM reviews WHERE review_id = ${review_id.reviews_id}`)
+    .then((reviewsIdData) => {
+      if (reviewsIdData.rows.length === 0) {
+        return Promise.reject({ status: 404, msg: "Not Found" });
+      } else {
+        let currentValue = reviewsIdData.rows[0][toPatchThisKey[0]];
```

Comment:
Can `currentValue` be a `const`

Reply:
Sadly not as the way I've gone to get the current value with the patch value. I'm reassigning the newValue = currentValue += patchValue so it needed to be a let. It was a const to start